src/modules/users.c

```diff
@@ -1,7 +1,15 @@
 #include "fastfetch.h"
 #include "common/printing.h"
 
-#include <utmpx.h>
+#if __has_include(<utmpx.h>)
+    #include <utmpx.h>
+#else
+    //for Android compatibility
+    #include <utmp.h>
+    #define utmpx utmp
+    #define setutxent setutent
+    #define getutxent getutent
+#endif
```

Comment:
I'd prefer to do that with cmakes `CHECK_INCLUDE_FILE`, too keep the non standard compliance low.